client/signup/steps/design-picker/index.jsx

```diff
@@ -126,18 +127,22 @@ class DesignPickerStep extends Component {
 	};
 
 	previewDesign = ( selectedDesign ) => {
+		const locale = ! this.props.userLoggedIn ? getLocaleSlug() : '';
+
 		recordTracksEvent( 'calypso_signup_design_preview_select', {
 			theme: `pub/${ selectedDesign.theme }`,
 			template: selectedDesign.template,
+			flow_name: this.props.flowName,
 		} );
 
-		page( getStepUrl( this.props.flowName, this.props.stepName, selectedDesign.theme ) );
+		page( getStepUrl( this.props.flowName, this.props.stepName, selectedDesign.theme, locale ) );
 	};
 
 	submitDesign = ( selectedDesign = this.state.selectedDesign ) => {
 		recordTracksEvent( 'calypso_signup_select_design', {
 			theme: `pub/${ selectedDesign?.theme }`,
 			template: selectedDesign?.template,
+			flow_name: this.props.flowName,
```

Comment:
I think we use `flow` to track the flowName

Reply:
Thanks, good catch! TIL!